src/question.py

```diff
@@ -0,0 +1,267 @@
+"""
+Question logic
+"""
+from __future__ import annotations
+
+
+import os
+import json
+import random
+import sys
+from dataclasses import dataclass
+from typing import ClassVar, List, Optional, Tuple
+import click
+from typing_extensions import TypeGuard
+
+from stats import Stats
+from translation import Translation
+
+
+@dataclass
+class Title:
+    """
+    Title class
+    """
+
+    principal: str
+    sub: List[str | None]
+
+
+@dataclass
+class Question:
+    """
+    Question class
+    """
+
+    __question_id: int
+    __title: Title
+    __propositions: List[str]
+    __responses: List[str]
+    __explication: str
+    __translation: Translation
+    __labels: ClassVar[Tuple[str, str, str, str]] = ("A", "B", "C", "D")
+
+    @property
+    def question_id(self) -> int:
+        """
+        Question id getter
+        """
+        return self.__question_id
+
+    def render(self) -> None:
+        """
+        Render cli question
+        """
+        self.__render_question()
+        self.__render_propositions()
+
+    def __render_propositions(self):
+        if len(self.__title.sub) > 0:
+            click.echo(self.__title.sub[0])
+        propositions = self.__propositions
+        max_proposition_len = len(propositions[0]) + 10
+        self.__render_proposition(self.__labels[:2], max_proposition_len, propositions[:2])
+        if len(propositions) > 2:
+            if len(self.__title.sub) > 1:
+                click.echo()
+                click.echo(self.__title.sub[1])
+            self.__render_proposition(self.__labels[2:], max_proposition_len, propositions[2:])
+        click.echo()
+
+    def __render_question(self):
+        click.echo("\n\n")
```

Comment:
I do not like this way of "forcing" spacing
But at this moment, I have no clue on how to do it in another way 😂